lib/Sabberworm/CSS/Value/Value.php

```diff
@@ -63,16 +63,25 @@ public static function parseValue(ParserState $oParserState, $aListDelimiters =
 	}
 
 	public static function parseIdentifierOrFunction(ParserState $oParserState, $bIgnoreCase = false) {
-		$sResult = $oParserState->parseIdentifier($bIgnoreCase);
+		$oParserState->setAnchor();
+		$mResult = $oParserState->parseIdentifier($bIgnoreCase);
 
 		if ($oParserState->comes('(')) {
-			$oParserState->consume('(');
-			$aArguments = Value::parseValue($oParserState, array('=', ' ', ','));
-			$sResult = new CSSFunction($sResult, $aArguments, ',', $oParserState->currentLine());
-			$oParserState->consume(')');
+			if ($oParserState->streql('url', $mResult)) {
+				$oParserState->backtrackToAnchor();
+				$mResult = URL::parse($oParserState);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'Sabberworm\CSS\Value\URL' in method 'parseIdentifierOrFunction'.](https://app.codacy.com/manual/sabberworm/PHP-CSS-Parser/pullRequest?prid=4484968)